dogen/templates/template.jinja

```diff
@@ -66,6 +66,8 @@ COPY *.rpm /tmp/rpms/
 # Install required RPMs
 RUN yum install -y {% if additional_repos -%} --disablerepo=\* {%- for repo in additional_repos %} --enablerepo={{ repo }}{% endfor %}{% endif %} {%- for package in packages %} {{ package }}{% endfor %} {%- for rpm in rpms %} /tmp/rpms/{{ rpm }} {% endfor %} \
     && yum clean all
+# Ensure that the packages were installed
+RUN rpm -q {% for package in packages %} {{ package }}{% endfor %}
```

Comment:
I would rather see this on same line as yum install - it will fail that layer and prevent it from being stored.